Tasks/NodeToolV0/task.json

```diff
@@ -13,8 +13,8 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 0,
-        "Minor": 166,
-        "Patch": 2
+        "Minor": 174,
+        "Patch": 0
```

Comment:
Sorry I missed this when it came in! This looks like a good change to me - would you mind bumping the version here and in the task.loc.json to 0.176.0? Once that happens I'll go ahead and merge it

Reply:
Anything I can do to get this through?